src/coreclr/src/jit/optcse.cpp

```diff
@@ -1729,23 +1729,17 @@ class CSE_Heuristic
             {
                 if (CodeOptKind() == Compiler::SMALL_CODE)
                 {
-                    moderateRefCnt = varDsc->lvRefCnt();
+                    moderateRefCnt = varDsc->lvRefCnt() + (BB_UNITY_WEIGHT / 2);
```

Comment:
Could you name these magic constants please? Also `moderateRefCnt` and `aggressiveRefCnt` could use a comment where they are declared. Right now I can't meaningfully review that, is it just tuning for CodeSize of System.Private.CoreLib?

Reply:
I second the request for now comments, and it would be good to understand the motivation behind these adjustments

Reply:
I will add lots more comments